suite/tests/client-interface/drwrap-drreg-test.dll.c

```diff
@@ -331,7 +351,11 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
     /* Look for nop;nop;nop in reg_val_test() and 4 nops in multipath_test(). */
```

Comment:
`reg_val_test()` and `multipath_test()` are an assembly function and they are currently not ported to RISC-V, maybe add a TODO here if this is a future work.

Reply:
I wrote assemblies, but it fails on assertion that check reg value restore functionality. I don't know whether it implemented, in this PR I supported only post-wrappers.